src/mod/endpoints/mod_sofia/rtp.c

```diff
@@ -129,20 +130,127 @@ switch_io_routines_t crtp_io_routines = {
 
 };
 
+#define CRTP_MEDIA_MODIFY_SYNTAX "<uuid> <remote ip> <remote port>"
+SWITCH_STANDARD_API(crtp_media_modify_function)
+{
+	switch_core_session_t *psession = NULL;
+    char *mycmd = NULL, *argv[3] = { 0 };
+	int argc = 0;
+
+	if (!zstr(cmd) && (mycmd = strdup(cmd))) {
+		argc = switch_separate_string(mycmd, ' ', argv, (sizeof(argv) / sizeof(argv[0])));
```

Comment:
We might need to check zstr(argv[0]) before line 141... I see that we've been checking it in line 142.

Reply:
argv supposted to be empty before line 41. The method `switch_separate_string` will perform tokenization and store the output to that array. Once this is executed, thats the time I validated the content of the array argv